src/mapeo-project.js

```diff
@@ -345,6 +346,20 @@ export class MapeoProject {
     await Promise.all([this.#coreManager.ready(), this.#ownershipWriteDone])
   }
 
+  /**
+   */
+  async close() {
+    this.#l.log('closing project %h', this.#projectId)
+    this.#coreManager.close()
```

Comment:
Better to close everything in parallel. The pattern I normally follow is to create an array like `const closePromises = []` then push each promise to that, then finally `await Promise.all(closePromises)`